src/suricata.c

```diff
@@ -1177,68 +1177,50 @@ static TmEcode ParseCommandLine(int argc, char** argv, SCInstance *suri)
     g_ut_covered = 0;
 #endif
 
-    struct option long_opts[] = {
-        {"dump-config", 0, &dump_config, 1},
-        {"dump-features", 0, &dump_features, 1},
-        {"pfring", optional_argument, 0, 0},
-        {"pfring-int", required_argument, 0, 0},
-        {"pfring-cluster-id", required_argument, 0, 0},
-        {"pfring-cluster-type", required_argument, 0, 0},
-        {"af-packet", optional_argument, 0, 0},
-        {"netmap", optional_argument, 0, 0},
-        {"pcap", optional_argument, 0, 0},
-        {"pcap-file-continuous", 0, 0, 0},
-        {"pcap-file-delete", 0, 0, 0},
-        {"pcap-file-recursive", 0, 0, 0},
-        {"simulate-ips", 0, 0 , 0},
-        {"no-random", 0, &g_disable_randomness, 1},
-        {"strict-rule-keywords", optional_argument, 0, 0},
-
-        {"capture-plugin", required_argument, 0, 0},
-        {"capture-plugin-args", required_argument, 0, 0},
+    struct option long_opts[] = { { "dump-config", 0, &dump_config, 1 },
+        { "dump-features", 0, &dump_features, 1 }, { "pfring", optional_argument, 0, 0 },
```

Comment:
I feel this is not an improvement, at all. Wonder if we can make the `.clang-format` file to not do this... @roligugus any ideas? Main complaint is not having each option on its own line.

Reply:
I couldn't find any options to fix this in the clang-format docs, so for now I've added `// clang-format off/on` around this section. I'm still reformatting the `if else if` stuff as it is being reformatted for the better.